### PR TITLE
Add a config docs template and update the guidelines

### PR DESCRIPTION
**Pull request by mmitoraj**
**Description**

Changes proposed in this pull request:

- Added a config docs template
- Updated the guidelines

**Related issue(s)**
See also #3636

### CODE REVIEW

File: guidelines/templates/resources/configuration.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+>* {[Sub-charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-sub-chart-overrides)}
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **{parameter}** | {Parameter description} | `{parameter value}` |
+| **{another.parameter}** | {Another parameter description} | `{another parameter value}` |
```
Comment (kazydek) [inline guidelines/templates/resources/configuration.md]: I am thinking if repeating it is required? 🤔

File: guidelines/templates/resources/configuration.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+type: Configuration
+---
+
+>**NOTE:** Blockquotes in this document provide instructions. Remove them from the final document.
+>
+>This document is a ready-to-use template for the **Configuration** document type that provides technical details about configuration of a Kyma chart or sub-chart. Follow the `05-{00}-{component-name}.md` convention to name the document. Make sure that the configuration document for the main chart displays first on the list. If there is only one document of a certain type, remove the `type` metadata completely so that the document displays well on the UI. 
+
+To configure the {Component name} {sub-}chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
+
+>Use the tip feature to provide links to the documentation about Kyma overrides. Add also a link to examples of either top-level charts overrides for charts configuration documents or sub-charts overrides for sub-chart configuration documents.
```
Comment (kazydek) [inline guidelines/templates/resources/configuration.md]:
```suggestion
>Use the tip feature to provide links to the documentation about Kyma overrides. Also add a link to examples of either top-level charts overrides for charts configuration documents or sub-charts overrides for sub-chart configuration documents.
```

Comment (kazydek) [reply guidelines/templates/resources/configuration.md]: Like here? https://github.com/YaaS/REST_API_Documentation_Guidelines/wiki/008-Also-Using-Also

File: guidelines/templates/resources/configuration.md (hunk 1 above)
Comment (mmitoraj) [reply guidelines/templates/resources/configuration.md]: Removed :)

**Pull request merged**